### DISCUSSION TITLE
Client Crashing

### DISCUSSION THREAD

**Discussion by Deacon207** [runelite/runelite#15576] [Q&A]
I bought a new computer and now my RL is constantly crashing at TOA. The screen freezes and all of RL becomes unresponsive. Anyone else had this issue?

**Comment by Ferrariic** [comment]
Can you post your client.log & crash logs `jvm_crash` might exist in your logs file iirc.

nvm saw that you did it in a separate issue: https://github.com/runelite/runelite/issues/15577
